core/src/main/scala-3/caliban/schema/SchemaDerivation.scala

```diff
@@ -48,9 +48,18 @@ trait SchemaDerivation[R] {
             true
           case _ => false
         }
+        lazy val isInterface = annotations.exists { case GQLInterface() =>
+          true
```

Comment:
Is it Scalafmt that forces this weird formatting? If you format it properly by hand, does the fmt check fail?

I should upgrade scalafmt to the latest version, not sure if it will help.

Reply:
Updated. I just thought that scalafmt would fix the formatting and forgot to check if it really did.